addons/edi_notifier/models/edi_document.py

```diff
@@ -21,6 +21,8 @@ class EdiDocumentType(models.Model):
 class EdiDocument(models.Model):
     _inherit = "edi.document"
 
+    notifier_subject_suffix = fields.Char("Suffix to the subject line of the notifier email")
```

Comment:
Nit: Should we capitalise the title - i.e. `Suffix to the Subject Line of the Notifier Email` ?

Reply:
Changed